docs/mirror.md

```diff
@@ -0,0 +1,175 @@
+# Create a local mirror of a Percona repository
+
+Enterprises introduce various security policies: for example, allow Internet access only on one server or deny root privileges to database administrators. 
+
+To comply with these policies and install Percona software in your environment, a solution is to set up the local mirror of Percona repositories. This document provides instructions how to do it.
+
+## Prerequisites
+
+To create a local mirror of a Percona repository, you need the following software:
+
+* Rsync to download packages from Percona repositories and upload them to your mirror
+* Repository creation tools:
+
+    - `dpkg-dev` for Debian and Ubuntu
+    - `createrepo` for RHEL and derivatives
+
+## On Debian and Ubuntu
+
+1. Install `dpkg-dev` utility to unpack, build and upload Debian source packages.
+
+    ```{.bash data-prompt="$"}
+    $ sudo apt install dpkg-dev
+    ```
+
+2. Create the folder where your mirror repository resides. For example, `/opt/debs/`
+
+    ```{.bash data-prompt="$"}
+    $ sudo  
+    ```
+
+3. Download the .deb packages relevant to your operating system from Percona repository into your repository using `rsync`:
+
+    ```{.bash data-prompt="$"}
+    $ rsync -avrt rsync://rsync.percona.com/rsync/<repo-name>/apt/pool/main/p/<product_name>/*.<os-version>*.deb /opt/debs/
+    ```
+
+    For example, to download Percona Backup for MongoDB packages on Ubuntu 22.04, the command is the following:
+
+    ```{.bash .no-copy}
+    $ rsync -avrt rsync://rsync.percona.com/rsync/pbm/apt/pool/main/p/percona-backup-mongodb/*.jammy*.deb /opt/debs/
+    ```
+
+    Check the list of available repositories for [MySQL](mysql.md), [MongoDB](mongodb.md), [PostgreSQL](postgresql.md) and [Percona Tools](tools.md).
+
+4. Create the Packages file. This is the file that the package manager uses to retrieve the information about new and updated packages when you run `apt-get update`. Use the `dpkg-scanpackages` command:
+
+    ```{.bash data-prompt="$"}
+    $ cd /opt/debs
+    $ dpkg-scanpackages . /dev/null > Packages
+    ```
+
+5. Add your repository to the `etc/apt/sources.list` file for the package manager to find new or updated packages from your repository:
+   
+    ```{.bash data-prompt="$"}
+    $ echo "deb [trusted=yes] file:///opt/debs ./" | sudo tee -a /etc/apt/sources.list
+    ```
```

Comment:
if we are downloading packages from our repository - the packages are signed with percona gpg key so we can import it and don't use [trusted=yes]. It will increase security

Reply:
Please advise the command how to do it

Reply:
curl -fsSL https://github.com/percona/percona-repositories/raw/main/deb/percona-keyring.gpg | sudo apt-key add -

or 
wget -qO- https://github.com/percona/percona-repositories/raw/main/deb/percona-keyring.gpg | sudo apt-key add -

Reply:
curl -fsSL https://github.com/percona/percona-repositories/raw/main/deb/percona-keyring.gpg | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/percona-keyring.gpg >/dev/null

Reply:
as for trusted=yes
if we remove it - apt will define this repository as unsigned and insecure - so it  will disable repo.
The solution is to create your own gpg key and sign repo with it or leave trusted=yes